src/pages/sites/[hash].tsx

```diff
@@ -28,6 +30,30 @@ export default function SitePage({ params }: IProps): JSX.Element {
 function SiteDetails({ site }: { site: GatsbySite }): JSX.Element {
   const { running, port, status } = useSiteRunnerStatus(site)
 
+  const version = site.gatsbyVersion
+
+  const adminSupported = useMemo(() => version && supportsAdmin(version), [
```

Comment:
Not sure `useMemo` is necessary here but probably doesn't hurt either 🤷

Reply:
I did it because semver.satisfies looks like it could be quite expensive. Includes object instantiation and lots of regex.